src/analytics/lambdas/custom-resource/create-schemas.ts

```diff
@@ -326,9 +326,9 @@ function getUpdatableSql(sqlOrViewDefs: SQLDef[], oldSqlArray: string[], mustach
   const updateFilesInfo = [];
   const sqlStatements: string[] = [];
   for (const schemaOrViewDef of sqlOrViewDefs) {
-    logger.info(`schemaOrViewDef.updatable: ${schemaOrViewDef.updatable}`);
+    logger.info(`schemaOrViewDef: sqlFile: ${schemaOrViewDef.sqlFile} - updatable: ${schemaOrViewDef.updatable}`);
 
-    if (!oldSqlArray.includes(schemaOrViewDef.sqlFile)) {
+    if (! isSqlFileInOldArray(schemaOrViewDef.sqlFile, oldSqlArray)) {
```

Comment:
So will it exclude the view definition for `clickstream_lifecycle_daily_view`? However, the view is required for reporting 1.1.0 based on new `event` table.

@llmin should we drop the old view if it exists when upgrading from 1.0.x if you are using the same view name?